### PR TITLE
Feat: Wishlist & Cart

### PR DESCRIPTION
**Pull request by Logan1x**
**Changes in this PR**
- Cart page added
- Wishlist Page added
- Favicon added in all pages using JS
- reduced repetitive CSS properties
- changed title of all the pages

**Deploy Preview**: 
- cart page: https://deploy-preview-4--plethora-project.netlify.app/pages/cart.html
- Wishlist page: https://deploy-preview-4--plethora-project.netlify.app/pages/wishlist.html

**ScreenShots:** 
- Wishlist

127 0 0 1_5500_pages_wishlist html

- Cart 

127 0 0 1_5500_pages_cart html

### CODE REVIEW

**Review by rohit8686 — APPROVED** [review]
Looks good !

File: styles/products.css [hunk 1]
```diff
@@ -89,7 +87,7 @@
   width: 25px;
```
Comment (Hnh013) [inline styles/products.css]: a suggestion : use 1.5 rem if possible in place of 25px

**Review by Hnh013 — APPROVED** [review]
one suggestion I've added , rest all is good !

File: styles/products.css (hunk 1 above)
Comment (Logan1x) [reply styles/products.css]: Resolved

**Comment by Logan1x** [comment]
Thanks @Hnh013 and @rohit8686 for review :)

**Pull request merged**